VersionControlUI/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ add_library(VersionControlUI SHARED
 	src/VersionControlUIException.h
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
+	src/DiffManager.h
```

Comment:
Each commit should at a minimum compile on its own. Here you add some source files in the CMakeLists.txt but don't add the actual files to the commit. This will fail to compile.